local/utils.inc.sh

```diff
@@ -148,6 +148,25 @@ function setup_git_repo() {
     popd
 }
 
+# usage: apt_get_retry <list of commands>
+function apt_get_retry() {
```

Comment:
I recommend adding some timeout for this loop to avoid infinite looping.

Reply:
When I will find a job that stops at the loop I will add this feature.